cypress/e2e/checkin/checkin.basic.cy.js

```diff
@@ -105,6 +104,9 @@ describe('Check In - Actions ', () => {
   });
 
   it('C347631 Check in: Basic check in (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {
+    cy.wait(10000);
```

Comment:
Why so long?

Reply:
it is 10 seconds. because regarding latest changes we need a time to synchronize with the page to be opened with all needed data.